<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Package Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Список пакетов</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchPackageName">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>570</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="placeholderText">
     <string>Введите название пакета</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_searchPackage">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>569</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="packageName">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>9</y>
      <width>361</width>
      <height>582</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="statusTip">
     <string>Здесь будет отображаться информация о выбранном пакете</string>
    </property>
    <property name="title">
     <string>Название пакета</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Команда для установки с помощью PIP:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="pipCommand">
        <property name="text">
         <string> </string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Описание:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Или с помощью автоматического установщика:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Версия:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="packageVersion">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="packageDescription">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="btn_installPackage">
        <property name="text">
         <string>Установить пакет</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Ссылка:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="packageHref">
        <property name="text">
         <string> </string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="btn_uninstallPackage">
        <property name="text">
         <string>Удалить пакет (если установлен)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QListWidget" name="packageList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>411</width>
      <height>531</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Здесь будут отображаться найденные пакеты</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchPackageName</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_searchPackage</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
